templates/static/form_handler.js

```diff
@@ -149,3 +171,50 @@ document.getElementById('url-form').addEventListener('submit', async function(ev
         }, 'image/png');
     });
 });
+
+async function downloadImage(imageElement, text, headlineElement, fontSize, color, fontType) {
```

Comment:
This function is duplicated. Remove it.

Reply:
I apologize for the oversight. I will remove the duplicated `downloadImage` function and ensure the existing one works correctly. I'll push the fix shortly.